events/app.py

```diff
@@ -69,7 +68,7 @@ def build_event_info(info, time):
 @app.route('/v1/edit/<event_id>', methods=['PUT'])
 def edit_event(event_id):
     """Edit the event with the given id."""
-    pass
+    pass    # pragma: no cover
```

Comment:
It's fine to leave these as tested by coverage -- it's a sign that something isn't entirely working. 😁